core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
I'm not sure we really decided on a name for the server side, the majority of the discussion was directed at the client.

I don't think the same argument for naming applies to the server. If we don't have a `Server`, why would we need a `ManagedServer`?